src/octoprint/plugins/classicwebcam/static/js/classicwebcam_wizard.js

```diff
@@ -0,0 +1,31 @@
+$(function () {
+    function CoreWizardWebcamViewModel(parameters) {
+        var self = this;
+
+        self.settingsViewModel = parameters[0];
+
+        self.required = false;
+
+        self.onWizardDetails = function (response) {
+            self.required =
+                response &&
+                response.corewizard &&
+                response.corewizard.details &&
+                response.corewizard.details.webcam &&
+                response.corewizard.details.webcam.required;
```

Comment:
I'm pretty sure that still needs adjustment.

Reply:
I'm bit at a loss here...is this actually needed? I think the entire callback can be removed as if I understand this code correctly is only required because CoreWizard offers multiple wizards. In our case here, if the wizard is launched it's always required.

Reply:
That's what I meant with adjustment ^^

Reply:
Will be removed :D